### ISSUE TITLE
HOI Annotation Method.

### ISSUE THREAD

**Issue by seasenzzr000** [opencv/cvat#7121]
I apologize for the inconvenience. I have searched through most articles, but I have not found any relevant documentation on CVAT for annotating Human-Object-Interactions (HOI). We are currently working on a project that requires annotating interactions between people and objects, but we have not been able to find specific guidance. Could you confirm if CVAT provides functionality for such annotations?

In simpler terms, a person may engage in interactions with multiple objects simultaneously (such as eating an apple, riding a bicycle, watching TV). It is necessary to establish corresponding relationships between the person and the objects and annotate them accordingly.

**Comment by bsekachev** [comment]
> It is necessary to establish corresponding relationships between the person and the objects and annotate them accordingly.

It is possible. For example define classes person and apple.
For class person attribute action: e.g. eating. 
Annotate them using necessary shapes and unite to group.

**Comment by seasenzzr000** [comment]
> > 需要建立人与物的对应关系，并进行相应的标注。
> 
> 有可能的。例如定义类 person 和 apple。 对于类人属性动作：例如吃。 使用必要的形状对它们进行注释并组合成组。

I apologize for any confusion or misunderstanding that may have occurred.A person may engage in multiple interactive actions simultaneously, such as looking, listening, reading, holding, stepping on, sitting, etc. Additionally, there may be a certain distance between the person and the interacted objects. How should these interactions be annotated?"

**Comment by bsekachev** [comment]
I am not sure distance plays role here. You may group objects anyway. 
With multiple interactive actions still possible to annotate, for example you have actions looking, listening, reading.
For class **person** you define three checkbox attributes with exactly same name and **false** value by default.
Then for each person you annotate these attributes, clicking **true** if person doing something.

But there may be problem with multiple relations between person and different subjects. Not clear for me how to understand that setup "eating" and "looking" attributes  refer to "apple" and "bread", not opposite. 

Maybe to simplify task. If person does 5 actions in parallel, you annotate this person 5 times. And one labeled object have only one attribute (as proposed initially) and grouped only with one interactable subject.
